lib/WP_Auth0_State_Handler.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+class WP_Auth0_State_Handler {
+
+  /**
+   * @var string
+   */
+  protected $uniqid;
+
+  /**
+   * @var string
+   */
+  const COOKIE_NAME = 'auth0_uniqid';
+
+  /**
+   * @var int
+   */
+  protected $cookieExpiresIn = MINUTE_IN_SECONDS;
```

Comment:
Where does this value comes from?

Reply:
WP core:

https://github.com/WordPress/WordPress/blob/master/wp-includes/default-constants.php#L129